docs/S3Installation.md

```diff
@@ -0,0 +1,20 @@
+# S3 Integration - Installing Boto
```

Comment:
why not add `boto` to `requirements.txt` file? Sounds easier than doing the installation from source, right?

Reply:
Good point, we made that modification and removed this S3Installation.md file entirely.